regex-parsing/src/test/resources/finders/VerboseRegexFinder.yml

```diff
@@ -0,0 +1,39 @@
+- 'features': ALL | ^BACKSLASH_ESCAPING
+
+- '[\s\S]': 's'       # Noncompliant {{Use concise character class syntax '.' instead of '[\s\S]'.}}
+- '[\d\D]'            # Noncompliant {{Use concise character class syntax '.' instead of '[\d\D]'.}}
+- '[\w\W]'            # Noncompliant {{Use concise character class syntax '.' instead of '[\w\W]'.}}
+- '[0-9]'             # Noncompliant {{Use concise character class syntax '\d' instead of '[0-9]'.}}
+- '[^0-9]'            # Noncompliant {{Use concise character class syntax '\D' instead of '[^0-9]'.}}
+- '[A-Za-z0-9_]'      # Noncompliant {{Use concise character class syntax '\w' instead of '[A-Za-z0-9_]'.}}
+- '[0-9_A-Za-z]'      # Noncompliant
+- '[^A-Za-z0-9_]'     # Noncompliant {{Use concise character class syntax '\W' instead of '[^A-Za-z0-9_]'.}}
+- '[^0-9_A-Za-z]'     # Noncompliant
+- 'x{0,1}'            # Noncompliant {{Use concise quantifier syntax '?' instead of '{0,1}'.}}
+- 'x{0,1}?'           # Noncompliant
+- 'x{0,}'             # Noncompliant {{Use concise quantifier syntax '*' instead of '{0,}'.}}
+- 'x{0,}?'            # Noncompliant
+- 'x{1,}'             # Noncompliant {{Use concise quantifier syntax '+' instead of '{1,}'.}}
+- 'x{1,}?'            # Noncompliant
```

Comment:
There!
The message expected is not quoted, but is it not true that /x{0,1}?/ is not the same as /x+/ ?
Or does the `?` in this case mean something else instead of optional?

Reply:
See lazy quantifiers